theano/tensor/basic.py

```diff
@@ -3055,7 +3055,9 @@ def var(input, axis=None, keepdims=False):
     centered_input = input - mean_input
 
     # return the mean sqr
-    return mean((centered_input ** 2), axis, keepdims=keepdims)
+    v = mean((centered_input ** 2), axis, keepdims=keepdims)
+    v.name = 'var'
+    return v
```

Comment:
I'll add the name in another PR later. I don't understand the correction needed in the docstring.